bin/oref0-setup.sh

```diff
@@ -1273,14 +1273,17 @@ if prompt_yn "" N; then
     # Install Golang
     mkdir -p $HOME/go
     source $HOME/.bash_profile
-    if go version | grep go1.11.; then
+    golangversion=1.12.5
+    if go version | grep go${golangversion}.; then
         echo Go already installed
     else
+        echo "Removing possible old go install..."
+        rm -rf /usr/local/go
```

Comment:
`rm -rf /usr/local/go/*` might work better here, as the directory itself can't be removed on Edisons with the FAT partition mounted at `/usr/local/go` (it produces a non-fatal error).

Reply:
fixed